digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/controller/SyncController.java

```diff
@@ -19,22 +21,23 @@
 @RequestMapping("/sync/v1")
 public class SyncController {
 
-    private final SyncLogService syncLogService;
+    private final SyncService syncLogService;
 
     @Autowired
-    public SyncController(SyncLogService syncLogService) {
+    public SyncController(FileSyncService syncLogService) {
```

Comment:
This should be SyncService syncLogService instead of FileSyncService

Reply:
Done